src/game/game.py

```diff
@@ -321,4 +321,5 @@ def main():
     game.run()
 
 
-main()
+if __name__ == '__main__':
```

Comment:
Pardon me if I'm missing a standard here, but in my code I have been using `"` instead of `'`, so we should probably stick with that for consistency.

Reply:
Totally reasonable. Will do.

If we want, we could add a `black` hook so we don't have to think about these things.